packages/camera/camera_android_camerax/pigeons/camerax_library.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(
+  PigeonOptions(
+    dartOut: 'lib/src/camerax_library.pigeon.dart',
+    dartTestOut: 'test/test_camerax_library.pigeon.dart',
+    dartOptions: DartOptions(copyrightHeader: <String>[
+      'Copyright 2013 The Flutter Authors. All rights reserved.',
+      'Use of this source code is governed by a BSD-style license that can be',
+      'found in the LICENSE file.',
+    ]),
+    javaOut:
+        'android/src/main/java/io/flutter/plugins/camerax/GeneratedCameraXLibrary.java',
+    javaOptions: JavaOptions(
+      package: 'io.flutter.plugins.camerax',
+      className: 'GeneratedCameraXLibrary',
+      copyrightHeader: <String>[
+        'Copyright 2013 The Flutter Authors. All rights reserved.',
+        'Use of this source code is governed by a BSD-style license that can be',
+        'found in the LICENSE file.',
+      ],
+    ),
+  ),
+)
+@HostApi(dartHostTestHandler: 'TestJavaObjectHostApi')
+abstract class JavaObjectHostApi {
+  void dispose(int instanceId);
```

Comment:
**small nit(optional):** the term `identifier` has personally worked better for us than `instanceId`. (since `instanceId` is close to `instance`.). We changed the term starting in the iOS implementation of the `webview_flutter` plugin.